doc/source/reference/fairseq2.nn.BatchLayout.rst

```diff
@@ -1,15 +1,10 @@
 .. _batch_layout:
 
-Batch Layout
-============
+fairseq2.nn.BatchLayout
```

Comment:
I assume this should also be `batch_layout` to be consistent with the rest of the API docs? (Also filename should be `fairseq2.nn.batch_layout` in that case)